src/main/java/com/maxmind/geoip/regionName.java

```diff
@@ -1,6 +1,11 @@
 package com.maxmind.geoip;
 // generated automatically from admin/generate_regionName.pl
-public class regionName {
+public final class regionName {
+
+    private regionName() throws InstantiationException {
+        throw new InstantiationException("This class is not for instantiation");
+    }
```

Comment:
This and the change to `timeZone` could break some integrations and will not be accepted. Also these two classes are automatically generated so changing them directly is pointless.

I'd be happy to merge the `LookupService` change though if the other two changes are removed.